src/Core/wwwroot/js/themeService.js

```diff
@@ -0,0 +1,32 @@
+export function switchHighlightStyle(dark) {
```

Comment:
Why not place this JS file under the razor file `FluentThemeProvider.razor` and only import it when the dev uses it.

Reply:
That's the perfect place to keep it. I'll make sure to move under the components tree